tools/aws_lab_setup/roles/control_node/templates/sshconfig.j2

```diff
@@ -0,0 +1,3 @@
+Host *
+     User ec2-user
```

Comment:
I'm pretty sure this break breaks centos support which doesn't have an ec2-user. @dfederlein am I mistaken?